tests/general/MementoBearTest.py

```diff
@@ -283,9 +284,6 @@ def test_settings_follow_redirects(self):
 
     def test_links_to_ignore(self):
         valid_file = """
-        # web.archive.org should be ignored
-        http://web.archive.org/web/*/spiegel.com
```

Comment:
because I've remove `DEFAULT_IGNORE` in `MementoBear` https://github.com/coala/coala-bears/pull/1889/files#diff-fe065107cab5427f5ff5a6a80e74788fL25